app/static/js/events/wizard/vue-app.js

```diff
@@ -350,10 +358,21 @@ function shouldDisableMove($this) {
         $this.event.start_time_time.trim() === '' ||
         $this.event.start_time_date.trim() === '' ||
         $this.event.end_time_time.trim() ==='' ||
-        $this.event.end_time_date.trim() === ''
+        $this.event.end_time_date.trim() === '' ||
+        checkTickets($this)
     );
 }
 
+function checkTickets($this) {
+    for( var ticket of $this.event.tickets){
+            if(ticket.type == 'paid' && (ticket.price === '' || ticket.price <=0))
+            {
```

Comment:
If one ticket is valid, why would the function exit?
if the ticket is valid, the `if` condition will not be satisfied, and the loop will keep moving forward.
Even if a single ticket is flawed then the function will exit and the button will be disabled.